setup.py

```diff
@@ -31,10 +31,11 @@
     "url": "https://github.com/neo4j-contrib/neo4j_doc_manager.git",
     "entry_points": {
         "console_scripts": [
+            'mongo-connector = mongo_connector.connector:main',
         ],
     },
     "packages": packages,
-    "install_requires": ["requests>=2.5.1"],
+    "install_requires": ["mongo-connector","requests>=2.5.1"],
```

Comment:
okay, seems like one that'll definitely need to be included (either via pip or running setup.py locally) -- but i'm okay approving this without it.